app/src/main/java/sw19/moring03/paint/Views/DrawingView.java

```diff
@@ -70,10 +72,15 @@ public boolean onTouchEvent(MotionEvent event) {
         return true;
     }
 
+    @Override
```

Comment:
only calls super pls get rid of it

Reply:
this was added because of a lint warning that wants you to override this function.